src/redis-helper.ts

```diff
@@ -119,11 +176,16 @@ export async function getTopTracksForArtist(redisClient: redis.RedisClient,
     const topTracksFromRedis: RedisTrack[]     = [];
     let   topTracksFromSpotify: SpotifyTrack[] = [];
 
-    if (!artist.top_track_ids || artist.top_track_ids.length === 0) {
-        // We've never gotten tracks and saved their ids for this artist, need to call spotify for tracks,
-        // save their ids for this artist, and save the tracks themselves. We cache all track IDs per artist
-        // and tracks themselves, but only return the number requested
-        console.log(`No top track ids for spotify artist ${
+    const lastUpdated = await getArtistTopTracksLastUpdated(redisClient, artist.id);
+
+    console.log(`Last updated top tracks: ${lastUpdated}`);
```

Comment:
I'd change this to only log if `lastUpdated` is missing, I don't think it should be logged every time.
```suggestion
    if (!lastUpdated) {
        console.log(`No last updated date for top tracks for artist ${artist.id} found`);
    }
```

Reply:
Done